digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1602,7 +1603,10 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     }
 
     split.flatMap(pl => {
-      val floatingValue = roadAddress.validTo.isDefined
+
+      // TODO If valid to is defined, what should be the floating reason? - Sami
```

Comment:
What should be the floating reason in this case?

Reply:
I think the reason being NewAddressGiven is not that bad.
Floating through:

ApplyChanges batch -> reason: ApplyChanges
Find Floating batch -> reason: GeometryChanged or NewAddressGiven, or GapInGeometry
Manualy by user in UI -> reason: ManualFloating
So i would say that we could use that NewAddressGiven since in fact we kind of create one new address road for each side of the cutting point (= 2 new roadaddresses) with spliting tool. Or do you think it would be better to have one new type to know were it was done through SplitingTool?